src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,5 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Not fixed. It's better to specify to which exactly tags you want to reset margin and padding

Reply:
done